test/tests.py

```diff
@@ -103,7 +104,7 @@ def setup_authentication(self, auth_type, username, password):
         return None
 
     def set_common_values(self):
-        self.index = f"my_index_{uuid.uuid4()}"
+        self.index = self.unique_id
```

Comment:
It seems like it would be nice to keep the previous format like: `f"test_index_{self.unique_id}"` to be able to clearly tell what our test index is

Reply:
I imagine our unique id for the integ tests may just be the name of the test and the pipeline run number like `full_run_21`